tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -18,8 +18,8 @@ jobs:
         script: |
           mkdir -p $HOME/.onnx
           docker run --rm --volume /data/onnx:/data/onnx:ro --volume $(Build.SourcesDirectory):/onnxruntime_src --volume $(Build.BinariesDirectory):/build --volume /data/models:/build/models:ro \
-          --volume $HOME/.onnx:/home/onnxruntimedev/.onnx -e NIGHTLY_BUILD onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimecpubuild:ch36 /opt/python/cp37-cp37m/bin/python3 \
-          /onnxruntime_src/tools/ci_build/build.py --build_dir /build --config Release --enable_lto \
+          --volume $HOME/.onnx:/home/onnxruntimedev/.onnx -e NIGHTLY_BUILD onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimecentoscpubuild:ch5g python3 \
+          /onnxruntime_src/tools/ci_build/build.py --build_dir /build --config Release \
```

Comment:
is removing enable_lto intentional?

Reply:
Yes. Otherwise the build will fail with a very strange error in mlas.